src/nrepl/jack-in.ts

```diff
@@ -108,6 +101,70 @@ export function calvaJackout() {
     liveShareSupport.didJackOut();
 }
 
+export async function copyJackInCommandToClipboard(): Promise<void> {
+    try {
+        await state.initProjectDir();
+    } catch (e) {
+        console.error('An error occurred while initializing project directory.', e);
+        return;
+    }
```

Comment:
This is done both here and in the jack-in process. Is there a reason to do it when jack-in is run and not earlier, like when Calva is activated?

Reply:
I don't quite recall, but I think it makes sense when you have several projects in the window. Then which one will be jacked-in to will be determined by the active editor, which we don't really know about at startup.